pyproject.toml

```diff
@@ -135,6 +136,9 @@ extend-exclude = [
 [tool.ruff.isort]
 combine-as-imports = true
 
+[tool.ruff.flake8-pytest-style]
+fixture-parentheses = false
```

Comment:
:tada:

I also like the setting that forces using a sequence of strings in parametrize rather than a comma-separated string...